client/my-sites/plan-features-2023-grid/_media-queries.scss

```diff
@@ -25,7 +36,16 @@ $minWidthToGridWidthMap: (
 			width: #{$gridWidth};
 		}
 	}
+}
+
+@mixin plan-comparison-grid-breakpoints {
```

Comment:
This is not related to your change but this name is a bit confusing. Because this is a mixing for `responsive container width` not breakpoints. But I am not sure i can come with a shorter name with explains this 😬